validphys2/src/validphys/core.py

```diff
@@ -140,7 +140,13 @@ def q_min(self):
     @property
     def error_type(self):
         """Error type as defined in the LHAPDF .info file"""
-        return self.info["ErrorType"]
+        try:
+            return self.info["ErrorType"]
+        except KeyError as e:
+            # If the error type is not defined _but_ the PDF only has one membe:
```

Comment:
Does this mean it's correctly interpreted as the cv everywhere down the line?

Reply:
Well, there's no error members so there's only the cv.

Reply:
I guess it could happen.

I'm actually adding one member (equal to the cv) to the LO PDF just to avoid that failure mode. Again, having just one PDF member doesn't seem to be a problem for validphys but there might be some weird function that rely on that.

Reply:
I think it was for the closure test that I created these `3replica` PDFs, where two error members were the same. Not sure anymore why this was necessary, I thought it might have been `np.std` that needed at least two inputs, but it seems to work with a single value as well.

Reply:
> I'm actually adding one member (equal to the cv) to the LO PDF just to avoid that failure mode.

I guess you're not planning to, but I wouldn't do this for the public version

Reply:
I was actually planning to. Why wouldn't you? The error, if you try to compute it, will be 0. But it won't fail if your program computes the error by default for instance.

Reply:
Because it risks confusing people. It's easy enough to make two copies of the `.dat` of rep0 that I don't think it will be much of an obstacle for people who's code relies on it (but if MC generators only load rep0 I guess that won't be a problem for the intended audience), and by forcing people to do it themselves they are very much aware of the hack. 

Then again, I also doubt it would really cause much confusion/harm...

Reply:
```
$ vp-pdffromreplicas --help
usage: vp-pdffromreplicas [-h] [--output-name OUTPUT_NAME] [--save-indices] input_pdf replicas

vp-pdffromreplicas Take a pre-existing source ``PDF`` with MC replicas and create a new ``PDF`` with replicas subsampled
from the source ``PDF``. Replicas will be sampled uniformly. source ``PDF`` will be downloaded if it cannot be found
locally. This script will not overwrite any existing files, so a ``PDF`` cannot already exist with the same name as the
output ``PDF``. Note that whilst in principle it is possible to create a single replica ``PDF`` whose replica 0 is simply
the same as replica 1, **LHAPDF has a restriction which requires ``PDF`` s to have at least 2 replicas (plus replica 0)**. To
handle this special case if ``replicas == 1``, then replica 2 will be a duplicate of replica 1, satisfying the minimum
number of replicas whilst retaining the property that replica 1 and replica 0 are identical.
```

Not sure if that's true? If not, we should maybe update it

Reply:
Ah I can't make something bold inside a code block. I mean this bit 
>LHAPDF has a restriction which requires ``PDF`` s to have at least 2 replicas (plus replica 0).

Reply:
That's strange, in the webpage it looks like the default is `1` https://lhapdf.hepforge.org/config.html so it would be weird to require 2.

Reply:
Yes I noticed that as well. Also the `Errormembers` key is optional. Maybe it's referring to the `LHAPFset` class or some other part of validphys? Would you mind updating it to say that some functions in vp require it instead of LHAPDF?